### DISCUSSION TITLE
Page build failed for the `master` branch with the following error:  You have an error on line 14 of your `_config.yml` file.

### DISCUSSION THREAD

**Discussion by yosemitefalls** [mmistakes/minimal-mistakes#2676] [Q&A] [answered]
https://github.com/yosemitefalls/yosemitefalls.github.io

howdy, I'm getting this error after pushing my changes. I followed the remote instructions and can't understand what's going wrong here.
any help would be appreciated, thank you

**Comment by mmistakes** [comment]
I do not recommend forking this repo to install, even if doing so using as a remote theme. There are many files you do not need that will end up giving you issues. 

Instead fork this starter repo. 
https://github.com/mmistakes/mm-github-pages-starter/generate

**Comment by mmistakes** [comment]
Also when I cloned your repo and built it locally I get the same error. Starting with a clean `_config.yml` it built fine. Not sure what the issue is but there's something in the YAML file perhaps that is causing a problem. Could be an improper indent or invalid character somewhere.

YAML is extremely unforgiving. Try running it through a YAML linter tool like http://www.yamllint.com/

My guess is there's something after line 14 that is invalid because if you remove all the lines after it, it passes.

**Comment by mmistakes** [comment, marked as answer]
Think these lines are your problem. The indentation is all wrong.
https://github.com/yosemitefalls/yosemitefalls.github.io/blob/master/_config.yml#L283-L288
